### ISSUE TITLE
OBS 28 Compatibility

### ISSUE THREAD

**Issue by leafac** [wtsnz/obs-ios-camera-source#84]
Hi @wtsnz,

OBS 28 is coming out soon and I’m thinking about updating. Before I do it, however, I want to check on the compatibility of the plugins that I use. Notably, OBS 28 updated their version of Qt, so plugin developers may have to do some extra steps to keep everything running.

How’s Camera for OBS Studio in that regard?

Thanks!

**Comment by wtsnz** [comment]
Hey @leafac 

I've been working on that this week - I expect to be able to get a build out that works with OBS 28.0 Beta 2 this evening :)

**Comment by leafac** [comment]
Awesome! Thank you very much for the work & for the quick answer!

**Comment by wtsnz** [comment]
Are you on Windows or MacOS?

Check out the latest [pre-release](https://github.com/wtsnz/obs-ios-camera-source/releases/tag/v2.9.2) for the build 👍

**Comment by leafac** [comment]
Great job! I’ll give it a try and report back. I’m on macOS M1 (the M1 native support is the reason why I’m looking into OBS 28 so early in the first place).

**Comment by wtsnz** [comment]
Excellent, macOS is working well from my tests! Let me know if you have any issues :)

**Comment by wtsnz** [comment]
OBS 28 is out now, and I've released the latest version of the plugin that works on macOS https://github.com/wtsnz/obs-ios-camera-source/releases/tag/v2.9.3

**Comment by leafac** [comment]
I upgraded to OBS 28.0.2 and obs-ios-camera-source 2.9.3, and something odd happened.

The installer (`.pkg`) put the plugin at `/Library/Application Support/obs-studio/plugins`, which used to be the correct location, but for whatever reason OBS didn’t pick it up. iOS Camera Sources weren’t available from the “Add” menu.

I noticed, however, that the [early release of the next version of obs-ndi](https://github.com/Palakis/obs-ndi/actions/runs/2977380826) placed itself on my home directory at `~/Library/Application Support/obs-studio/plugins`.

So I moved obs-ios-camera-source there and it worked 🤷
